calculator/src/main/java/edu/nextstep/camp/calculator/CalculatorViewModel.kt

```diff
@@ -0,0 +1,47 @@
+package edu.nextstep.camp.calculator
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import edu.nextstep.camp.calculator.domain.Calculator
+import edu.nextstep.camp.calculator.domain.Expression
+import edu.nextstep.camp.calculator.domain.Operator
+
+class CalculatorViewModel(
+    private var expression: Expression = Expression.EMPTY,
+    private val calculator: Calculator = Calculator()
+) : ViewModel() {
+
+    private val _showingExpression = MutableLiveData<String>()
+    val showingExpression: LiveData<String>
+        get() = _showingExpression
+
+    private val _calculatorError = SingleLiveEvent<Event<Unit>>()
+    val calculatorError: LiveData<Event<Unit>> = _calculatorError
+
+    fun addToExpression(operand: Int) {
+        expression += operand
+        _showingExpression.value = expression.toString()
```

Comment:
```suggestion
        _showingExpression.value = expression.toString()
```
해당 코드가 여러 함수에서 중복으로 사용하고 있네요.
함수 분리를 통해서 중복 코드를 줄여보면 어떨까요?